community/inventory_service_tags_italian.txt

```diff
@@ -3,7 +3,7 @@ rarity		Rarità
 	uncommon		Non comune
 	rare		Raro
 	legendary		Leggendario
-	genuine		
-	strange		
+	genuine		Genuino
+	strange		Strano
 slot		
```

Comment:
in case you missed it, "slot" is also a translation key

Reply:
oh!